### ISSUE TITLE
Error processing algorithm in Processing-Taudem

### ISSUE THREAD

**Issue by romeuvieira** [alexbruy/processing-taudem#13]
Hi,

I am a user of QGIS, trying to use "processing-taudem" plugin. I think I have followed all instructions to correctely intal it bu in all commands I get the same type of error. So, basically I have to use the old Wien version to run Taudem.

This is part of the meaase-error I get:

_.../AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_taudem\taudemAlgorithm.py", line 54, in processAlgorithm
command = generateCommand(self, parameters, context, feedback)
NameError: name 'generateCommand' is not defined_

Is this enough to someone with expertise to help me?

**Comment by alexbruy** [comment]
Not reproducible with current version.

**Comment by romeuvieira** [comment]
Thanks.

So it don´t works with 3.12. What would be the one that it will run? 3.10?

**Comment by alexbruy** [comment]
It **does** work in any QGIS 3.x.

**Comment by romeuvieira** [comment]
> Not reproducible with current version.

So, you where refering to the Wien one?

I am trying to work currently with the processing_taudem plugin on vesion 3.12. But getting the error message above.

**Comment by yhan27** [comment]
Hi, I ended up with the same error message after I successfully installed the plug in and TauDEM executive files. thought? Error Message below after I tried to run pit remove function from QGIS 3.8.3, 64bit, windows 10.

Traceback (most recent call last):
File "C:/Users/myname/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_taudem\taudemAlgorithm.py", line 54, in processAlgorithm
command = generateCommand(self, parameters, context, feedback)
NameError: name 'generateCommand' is not defined

Execution failed after 0.04 seconds
